apostrophecms/apostrophe
# support A3 projects with npm workspaces packages

## Summary

Add support for direct workspace dependencies

## What are the specific steps to test this change?

You run the test `test/workspaces-project.js` or

1. Start from `a3-boilerplate`
2. Run `npm init -w workspace-a -y` to create `workspace-a`
3. Install `@apostrophecms-pro/scheduled-publishing` in `workspace-a` with `npm install -w workspace-a --save @apostrophecms-pro/scheduled-publishing`
4. In the `a3-boilerplate` project, enable the module `@apostrophecms-pro/scheduled-publishing`. It should not be present in the `a3-boilerplate` `package.json`. It should pass.
5. Switch to `apostrophe` `main` branch, the same thing should fail with `The configured bundle @apostrophecms-pro/scheduled-publishing was not found in npm`

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
